### ISSUE TITLE
TypeScript's built-in "inline variable" refactoring should be declared as an overlapping code assist

### ISSUE THREAD

**Issue by OliverJAsh** [p42ai/refactor-vscode#74]
https://devblogs.microsoft.com/typescript/announcing-typescript-5-2-beta/#inline-variable-refactoring

I think we need to define this as an overlapping code assist for `inline-variable`. Is that correct?

<img width="506" alt="image" src="https://github.com/p42ai/js-assistant/assets/921609/f043e569-e5b9-4cc4-a775-522e66486423">

**Comment by lgrammel** [comment]
Yes, thanks for bringing this up. Not sure when I'll have time to look into this - the hidden typescript refactorings are configured here if you want to take a stab at it:

https://github.com/p42ai/js-assistant/blob/main/app/vscode-refactor/hide-typescript-refactors/src/index.ts
